gobblin-service/src/main/java/org/apache/gobblin/service/modules/restli/GobblinServiceFlowConfigResourceHandler.java

```diff
@@ -96,6 +97,12 @@ public CreateResponse createFlowConfig(FlowConfig flowConfig)
     String flowName = flowConfig.getId().getFlowName();
     String flowGroup = flowConfig.getId().getFlowGroup();
 
+    if (flowConfig.getProperties().containsKey(ConfigurationKeys.FLOW_EXECUTION_ID_KEY)) {
```

Comment:
Will this be executed only in master mode?

Reply:
This will be executed on the node which receives the request. And this happens once. This then forward to master (other or itself) local resource handler.